newsrevenuehub/rev-engine
# refs #NRH-172 Adds React Sentry Tracing.

**What does this PR do?**
It adds Sentry browser tracing to the React App. 

**How should this be manually tested?**
This is difficult to test live. What I did locally, was add a <NavItem> to the dashboard that, when clicked, throws an error.
If you wish to test on a live server, you could use the site until a bug is found :smile:, or add a commit with the added code to test then remove.
Sentry suggests `return <button onClick={methodDoesNotExist}>Break the world</button>;` should work, but it did not for me in a local context from the console.

**Things to note**
This is the first project that we have split the reporting. Currently we have reporting from Django going to one DSN, and now with this PR reporting from the SPA is being routed to another DSN.  This is not strictly necessary but it might help with separation of concerns.
